tutor/src/components/new-course/wizard.cjsx

```diff
@@ -4,16 +4,19 @@ BS = require 'react-bootstrap'
 classnames = require 'classnames'
 keys = require 'lodash/keys'
 
+TutorRouter = require '../../helpers/router'
+
 BindStore = require '../bind-store-mixin'
 {NewCourseActions, NewCourseStore} = require '../../flux/new-course'
+{CourseActions, CourseStore} = require '../../flux/course'
 
 STAGES = {
-  'course_type': require './select-type'
-  'offering_id': require './select-course'
-  'term':        require './select-dates'
-  'details':     require './course-details'
-  'copy_ql':     require './copy-ql'
-  'build':       require './build-course'
+  'course_type':              require './select-type'
+  'offering_id':              require './select-course'
+  'term':                     require './select-dates'
+  'details':                  require './course-details'
+  'copy_question_library':    require './copy-ql'
+  'build':                    require './build-course'
 }
```

Comment:
Nice catch, I'd had the key named `copy_ql` everywhere, then the BE named it as the longer version, looks like I missed renaming in some spots.